controllers/configurationpolicy_utils.go

```diff
@@ -198,11 +198,8 @@ func checkFieldsWithSort(mergedObj map[string]interface{}, oldObj map[string]int
 			mQty, err := apiRes.ParseQuantity(mVal)
 			if err != nil {
 				oVal := oldObj[i]
-				if oVal == nil {
```

Comment:
Why did you remove this?

Reply:
When the label that we want to add doesn't exist in the current object, `oVal` is set to `nil`. If we set `oVal` to an empty string here, the comparison `oVal != mVal` will always evaluate to `false` when `mVal` is set to an empty string as well. So the controller never emits an update event on the object. I think it's important to retain the  `nil` value as a way to detect fields that don't exist in the current object.

Reply:
@JustinKuli Would you help review this?

Reply:
I think I'd prefer
```
				oVal, keyPresent := oldObj[i]
				if !keyPresent {
					return false
				}
```

But I think either would work.